opentech/apply/dashboard/views.py

```diff
@@ -23,34 +26,88 @@ def get(self, request, *args, **kwargs):
         closed_query = '?round_state=closed'
         rounds_title = 'Your rounds and labs'
 
+        my_review_qs = qs.in_review_for(request.user).order_by('-submit_time')
+        my_review = SummarySubmissionsTable(my_review_qs[:5], prefix='my-review-')
+        display_more = (my_review_qs.count() > 5)
+
+        # Staff reviewer's reviewed submissions for 'Previous reviews' block
+        # Replicating django_filters.views.FilterView
+        my_reviewed_qs = qs.reviewed_by(request.user).order_by('-submit_time')
+        kwargs = {
+            'data': self.request.GET or None,
+            'request': self.request,
+        }
+        kwargs.update({
+            'queryset': my_reviewed_qs,
+        })
+        self.filterset = SubmissionFilterAndSearch(**kwargs)
+        my_reviewed_qs = self.filterset.qs
+
+        my_reviewed = SummarySubmissionsTable(my_reviewed_qs[:5], prefix='my-reviewed-')
+        display_more_reviewed = (my_reviewed_qs.count() > 5)
+
         return render(request, 'dashboard/dashboard.html', {
-            'in_review': in_review,
             'open_rounds': open_rounds,
             'open_query': open_query,
             'closed_rounds': closed_rounds,
             'closed_query': closed_query,
             'rounds_title': rounds_title,
+            'my_review': my_review,
+            'in_review_count': my_review_qs.count(),
+            'display_more': display_more,
+            'my_reviewed': my_reviewed,
+            'display_more_reviewed': display_more_reviewed,
+            'filter': self.filterset,
         })
 
 
 class ReviewerDashboardView(TemplateView):
+
     def get(self, request, *args, **kwargs):
         qs = ApplicationSubmission.objects.all().for_table(self.request.user)
 
-        my_review_qs = qs.in_review_for(request.user)
-        my_review = SubmissionsTable(my_review_qs, prefix='my-review-')
-        RequestConfig(request, paginate={'per_page': 10}).configure(my_review)
+        # Reviewer's current to-review submissions
```

Comment:
can we pull some of this out into some methods just to tidy up what is going on.